ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarGroupDemoController.swift

```diff
@@ -0,0 +1,262 @@
+
+        maxAvatarButton.view.widthAnchor.constraint(equalToConstant: 50).isActive = true
+        addRow(text: "Max displayed avatars", items: [UIStackView(arrangedSubviews: [maxAvatarsTextField, maxAvatarButton.view])], textStyle: .footnote, textWidth: Constants.settingsTextWidth)
+        overflowCountButton.view.widthAnchor.constraint(equalToConstant: 50).isActive = true
+        addRow(text: "Overflow count", items: [UIStackView(arrangedSubviews: [overflowCountTextField, overflowCountButton.view])], textStyle: .footnote, textWidth: Constants.settingsTextWidth)
+
+        addTitle(text: "Avatar Stack No Border")
+        insertAvatarViews(style: .stack, showBorders: false)
+        addTitle(text: "Avatar Pile No Border")
+        insertAvatarViews(style: .pile, showBorders: false)
+    }
+
+    private struct Constants {
+        static let settingsTextWidth: CGFloat = 200
+        static let avatarsTextWidth: CGFloat = 100
+        static let maxTextInputCharCount: Int = 4
+    }
+
+    private var avatarGroups: [MSFAvatarGroup] = []
+
+    private var avatarCount: Int = 5 {
+        didSet {
+            reloadAvatarViews()
+        }
+    }
+
+    private var isUsingAlternateBackgroundColor: Bool = false {
+        didSet {
+            updateBackgroundColor()
+        }
+    }
+
+    private func updateBackgroundColor() {
+        view.backgroundColor = isUsingAlternateBackgroundColor ? UIColor(light: Colors.gray100, dark: Colors.gray600) : Colors.surfacePrimary
+    }
+
+    @objc private func toggleAlternateBackground(switchView: UISwitch) {
+        isUsingAlternateBackgroundColor = switchView.isOn
+    }
+
+    private var maxDisplayedAvatars: UInt = 4 {
+        didSet {
+            if oldValue != maxDisplayedAvatars {
+                maxAvatarsTextField.text = "\(maxDisplayedAvatars)"
+
+                for avatarGroup in avatarGroups {
+                    avatarGroup.state.maxDisplayedAvatars = UInt32(maxDisplayedAvatars)
+                }
+            }
+        }
+    }
+
+    private var overflowCount: UInt = 0 {
+        didSet {
+            if oldValue != overflowCount {
+                overflowCountTextField.text = "\(overflowCount)"
+
+                for avatarGroup in avatarGroups {
+                    avatarGroup.state.overflowCount = overflowCount
+                }
+            }
+        }
+    }
+
+    private func insertAvatarViews(style: MSFAvatarGroupStyle, showBorders: Bool, mixed: Bool = false) {
+        var constraints: [NSLayoutConstraint] = []
+
+        for size in MSFAvatarSize.allCases.reversed() {
+            let containerView = UIView(frame: .zero)
+
+            let avatarGroup = MSFAvatarGroup(style: style, size: size)
+            var border = showBorders
+            for index in 0...avatarCount - 1 {
+                vSamplePersonas[index].isRingVisible = border
```

Comment:
We could showcase the ability to set individual border settings to avatars in the control (as this is one of the scenarios that will be required for the AvatarGroup).